README.md

```diff
@@ -1,163 +1,28 @@
+Example use-cases are provided on the orb [registry page](https://circleci.com/orbs/registry/orb/circleci/aws-s3#usage-examples). Source for these examples can be found within the `src/examples` directory.
 
 
-### Permissions
+## Resources
 
-Explanation of all permissions required for the script.
+[CircleCI Orb Registry Page](https://circleci.com/orbs/registry/orb/circleci/aws-s3) - The official registry page of this orb for all versions, executors, commands, and jobs described.  
+[CircleCI Orb Docs](https://circleci.com/docs/2.0/orb-intro/#section=configuration) - Docs for using and creating CircleCI Orbs.  
 
-* **sudo** - The CircleCI CLI Update command will request sudo permissions to update.
+AWS S3 Docs: https://docs.aws.amazon.com/s3/index.html
 
-### Preview
+### How To Contribute
 
-<details>
-<Summary>Preview "Hello-World" Orb produced by this repo by default.</Summary>
+We welcome [issues](https://github.com/CircleCI-Public/aws-s3-orb/issues) to and [pull requests](https://github.com/CircleCI-Public/aws-s3-orb/pulls) against this repository!
 
-```yaml
-commands:
-  greet:
-    description: |
-      Replace this text with a description for this command. # What will this command do? # Descriptions should be short, simple, and clear.
-    parameters:
-      greeting:
-        default: Hello
-        description: Select a proper greeting
-        type: string
-    steps:
-    - run:
-        command: echo << parameters.greeting >> world
-        name: Hello World
-description: |
-  Sample orb description # What will your orb allow users to do? # Descriptions should be short, simple, and clear.
-examples:
-  example:
-    description: |
-      Sample example description. # What will this example document? # Descriptions should be short, simple, and clear.
-    usage:
-      jobs:
-        build:
-          machine: true
-          steps:
-          - foo/hello:
-              username: Anna
-      orbs:
-        foo: bar/foo@1.2.3
-      version: 2.1
-executors:
-  default:
-    description: |
-      This is a sample executor using Docker and Node. # What is this executor? # Descriptions should be short, simple, and clear.
-    docker:
-    - image: circleci/node:<<parameters.tag>>
-    parameters:
-      tag:
-        default: latest
-        description: |
-          Pick a specific circleci/node image variant: https://hub.docker.com/r/circleci/node/tags
-        type: string
-jobs:
-  hello:
-    description: |
-      # What will this job do? # Descriptions should be short, simple, and clear.
-    executor: default
-    parameters:
-      greeting:
-        default: Hello
-        description: Select a proper greeting
-        type: string
-    steps:
-    - greet:
-        greeting: << parameters.greeting >>
-orbs:
-  hello: circleci/hello-build@0.0.5
-version: 2.1
-```
+To publish a new production version:
```

Comment:
This text seems to be from the orb starter kit, does it still need to be included?

Reply:
👍 I wanted to include this as it's still valid documentation for us as maintainers (future documentation) and would help anyone who might fork it.